tests/Feature/Leads/LeadsTest.php

```
@@ -0,0 +1,652 @@
+            'status_id'        => 999,
+        ]);
+
+        /* Assert */
+        $this->lead->refresh();
+
+        $response->assertStatus(302);
+        $this->assertEquals($newUser->id, $this->lead->user_assigned_id);
+        $this->assertEquals($originalTitle, $this->lead->title);
+        $this->assertEquals($originalDescription, $this->lead->description);
+        $this->assertNotEquals(999, $this->lead->status_id);
+    }
```

PHPMD (2.15.0)

[error] The method it_lead_update_assign_only_accepts_user_assigned_id_field is not named in camelCase. (undefined)

(CamelCaseMethodName)